lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -954,7 +954,7 @@ public function setValue(string $column, string $value) : self
      *         );
      * </code>
      *
-     * @param array<int, mixed> $values The values to specify for the insert query indexed by column names.
+     * @param array<string, mixed> $values The values to specify for the insert query indexed by column names.
```

Comment:
It looks like `$values` is a list (i.e. the keys are not used):

https://github.com/doctrine/dbal/blob/8b373f4a5ba339be6c0740256dfa271fca34f8d6/lib/Doctrine/DBAL/Query/QueryBuilder.php#L1191-L1193

Please double-check if this change is needed.

Reply:
I'm not following you, the snippet you mention is not where the next call jumps to (`add()`)?

And the change is in line with the description:

> $values The values to specify for the insert query **indexed by column names**.

Reply:
Oh, sorry, I didn't see that there was `array_keys()` used on values. You're right.